### PR TITLE
XW-1877 | Search suggestions in Global Navigation

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/XW-1877
https://github.com/Wikia/design-system/pull/43

### CODE REVIEW

File: components/global-navigation.md [hunk 1]
```diff
@@ -397,3 +397,98 @@
...
+						<button class="wds-button wds-is-text wds-global-navigation__search-close" type="button">
+							<svg class="wds-icon wds-icon-small wds-global-navigation__search-close-icon" alt="Close">
+								<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-icons-cross"></use>
+							</svg>
+						</button>
+						<div class="wds-dropdown-content wds-global-navigation__search-suggestions">
+							<ul class="wds-list">
+								<li class="wds-global-navigation__search-suggestion">
+									<a href="#" class="wds-global-navigation__dropdown-link">
+										<span><strong>Star</strong>&nbsp;Wars</span>
```
Comment (rogatty) [inline components/global-navigation.md]: Styledown removes "unnecessary" spaces and that's why I needed to use `&nbsp;` here.

Comment (kvas-damian) [reply components/global-navigation.md]: Why do we need a `span` here?

File: components/global-navigation.md [hunk 2]
```diff
@@ -397,3 +397,98 @@
...
+							<input class="wds-global-navigation__search-input" name="search" placeholder="Szukaj na Wookieepedia..." value="Star"/>
+						</label>
+						<button class="wds-button wds-is-text wds-global-navigation__search-close" type="button">
+							<svg class="wds-icon wds-icon-small wds-global-navigation__search-close-icon" alt="Close">
+								<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#wds-icons-cross"></use>
+							</svg>
+						</button>
+						<div class="wds-dropdown-content wds-global-navigation__search-suggestions">
+							<ul class="wds-list">
+								<li class="wds-global-navigation__search-suggestion">
```
Comment (kvas-damian) [inline components/global-navigation.md]: I remember @tomasznapieralski decided to not add class name to every element in a list few days ago. Can we make sure we are consistent in what we do?

File: components/global-navigation.md (hunk 1 above)
Comment (rogatty) [reply components/global-navigation.md]: We don't.

**Comment by Warkot** [comment]
You might want to add a custom class which adds the margin under the nav which has opened search suggestions dropdown, so the dropdown doesn't cover other content.

Like here https://github.com/Wikia/design-system-reference-page/blob/gh-pages/config/custom-styles.css#L77-L80

**Comment by hakubo** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**